src/contributors/02-local-development.md

```diff
@@ -82,7 +82,7 @@ After making changes, you need to format the code with `cargo +nightly fmt --all
 
 ### Frontend development
 
-Install dependencies by running `yarn`. Then run `yarn start` to launch lemmy-ui locally. It automatically connects to the Lemmy backend on `localhost:8536`. Open [localhost:1234](http://localhost:1234) in your browser. The project is rebuilt automatically when you change any files.
+Install dependencies by running `yarn`. Then run `yarn build:dev --watch` to launch lemmy-ui locally. It automatically connects to the Lemmy backend on `localhost:8536`. Open [localhost:1234](http://localhost:1234) in your browser. The project is rebuilt automatically when you change any files.
```

Comment:
It seems like `yarn dev` is a short-hand version of this, which might make it a slight bit easier

Reply:
You're right, I'll change it.